pages/common/distccd.md

```diff
@@ -0,0 +1,24 @@
+# distccd
+
+> Server daemon for distcc distributed compiler.
+> More information: <https://distcc.github.io>.
+
+- Start a daemon with default settings:
+
+`distccd --daemon`
+
+- Start a daemon, accepting connections from private network IPv4 ranges:
+
+`distccd --daemon --allow-private`
+
+- Start a daemon, accepting connections from a specific network address or address range:
+
+`distccd --daemon --allow {{ip_address|network_prefix}}`
+
+- Start a daemon with a maximum of 4 tasks at any time and lowered priority (niceness = 5):
+
+`distccd --daemon --jobs {{4}} --nice {{5}}`
```

Comment:
Suggested modification for a point risen by @marchersimon:

```suggestion
- Start a daemon with a maximum of 4 tasks at any time and the lowest priority (niceness = 19):

`distccd --daemon --jobs {{4}} --nice {{19}}`
```

Reply:
I really don't think a novice user needs to know all that. Any priority lower than normal will be fine for most user cases. How often do you adjust priority of the processes anyway? ;)

Reply:
Since this change turns out to complicate the example quite a bit, I think we should keep the original.